packages/payload/src/preferences/requestHandlers/update.ts

```diff
@@ -5,6 +5,19 @@ import type { PayloadHandler } from '../../config/types.js'
 import update from '../operations/update.js'
 
 export const updateHandler: PayloadHandler = async (req) => {
+  let data
+
+  try {
+    data = await req.json()
+  } catch (error) {
+    data = {}
+  }
+
+  if (data) {
+    req.data = data
+    req.json = () => Promise.resolve(data)
+  }
+
```

Comment:
It might be nice to use the utilities, would serve as an example of usage in case stumbled upon. Thoughts?

This would apply to all preferences endpoints

Reply:
We can't import from the next package in the payload package